packages/hurl/src/cli/options/mod.rs

```diff
@@ -92,6 +92,7 @@ pub struct CliOptions {
     pub variables: HashMap<String, Value>,
     pub verbose: bool,
     pub very_verbose: bool,
+    pub max_filesize: usize,
```

Comment:
Maybe a better type would be Option<max_filesize>, it will enable not to check against 0